### PR TITLE
remove include/GL/wglext.h from Mesa sanity check

### PR DESCRIPTION
**Pull request by damianam**
Fixes https://github.com/hpcugent/easybuild-easyconfigs/issues/4352

### CODE REVIEW

File: easybuild/easyconfigs/m/Mesa/Mesa-10.4.5-intel-2015a-Python-2.7.9.eb [hunk 1]
```diff
@@ -64,9 +64,8 @@ prebuildopts = 'env CPATH="$EBROOTLIBDRM/include/libdrm:$CPATH" '
 sanity_check_paths = {
     'files': ['lib/libGL.%s' % SHLIB_EXT, 'include/GL/glext.h', 'include/GL/gl_mangle.h',
               'include/GL/glx.h', 'include/GL/osmesa.h',
-              'include/GL/wglext.h', 'include/GL/gl.h',
-              'include/GL/glxext.h', 'include/GL/glx_mangle.h',
-              'include/GL/wmesa.h'],
+              'include/GL/gl.h', 'include/GL/glxext.h',
+              'include/GL/glx_mangle.h', ^'include/GL/wmesa.h'],
```
Comment (boegel) [inline easybuild/easyconfigs/m/Mesa/Mesa-10.4.5-intel-2015a-Python-2.7.9.eb]: `^` causes syntax error

File: easybuild/easyconfigs/m/Mesa/Mesa-10.5.5-intel-2015a-Python-2.7.10.eb [hunk 2]
```diff
@@ -64,9 +64,8 @@ sanity_check_paths = {
     'files': ['lib/libGL.%s' % SHLIB_EXT, 'lib/libOSMesa.%s' % SHLIB_EXT,
               'include/GL/glext.h', 'include/GL/gl_mangle.h',
               'include/GL/glx.h', 'include/GL/osmesa.h',
-              'include/GL/wglext.h', 'include/GL/gl.h',
-              'include/GL/glxext.h', 'include/GL/glx_mangle.h',
-              'include/GL/wmesa.h'],
+              'include/GL/gl.h', 'include/GL/glxext.h',
+              'include/GL/glx_mangle.h', ^'include/GL/wmesa.h'],
```
Comment (boegel) [inline easybuild/easyconfigs/m/Mesa/Mesa-10.5.5-intel-2015a-Python-2.7.10.eb]: `^` causes syntax error

File: easybuild/easyconfigs/m/Mesa/Mesa-10.4.5-intel-2015a-Python-2.7.9.eb (hunk 1 above)
Comment (damianam) [inline easybuild/easyconfigs/m/Mesa/Mesa-10.4.5-intel-2015a-Python-2.7.9.eb]: Argh! Sorry, it should be fixed now.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2589.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/dfd3c886f64aa0921ef3871e3c7f4adb for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2128.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2b59d102c2f709436d076a6c6c605e31 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @damianam!

**Pull request merged**